tests/conftest.py

```diff
@@ -151,7 +151,7 @@ def chdir_tmp_path(tmp_path):
 # CMC 2024-03-01 this doesn't need to be a fixture
 @pytest.fixture
 def keccak():
-    return Web3.keccak
+    return keccak256
```

Comment:
honestly we should just remove this as a fixture -- test modules can import `vyper.utils.keccak256()`

Reply:
we should probably also have a sanity check unit test, like for the constant `keccak256(b"")` case should return the well known result c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470 (https://ethereum.stackexchange.com/a/107985)

Reply:
err wrong PR!